tests/python/pants_test/engine/legacy/test_parser.py

```diff
@@ -16,7 +16,7 @@ class LegacyPythonCallbacksParserTest(unittest.TestCase):
 
   def test_no_import_sideeffects(self):
     # A parser with no symbols registered.
-    parser = LegacyPythonCallbacksParser(EmptyTable(), BuildFileAliases())
+    parser = LegacyPythonCallbacksParser(EmptyTable(), BuildFileAliases(), 'allow')
```

Comment:
nit: could you use the kwarg here? I think `build_file_imports_behavior='allow'` would be a lot clearer to someone looking at this later.